support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -51,6 +51,7 @@ import GeneralErrorMessage from 'components/generalErrorMessage/generalErrorMess
 import { toHumanReadableContributionType, getAvailablePaymentRequestButtonPaymentMethod } from 'helpers/checkouts';
 import type { Option } from 'helpers/types/option';
 import type { Csrf as CsrfState } from '../../../../helpers/csrf/csrfReducer';
+import {trackComponentEvents} from "../../../../helpers/tracking/ophan";
```

Comment:
Just a tiny style one - looks like we add spaces in curly braces i.e

```js
import { trackComponentEvents } from "../../../../helpers/tracking/ophan";
```

Reply:
Good spot, cheers @TomPretty! I've sorted this now ✅